README.md

```diff
@@ -1,33 +1,64 @@
+**Docker**
+
+We use Docker to make installation quick and painless, [make sure you have it installed](https://docs.docker.com/engine/installation/). This has been tested to work with at least `docker 17.03.0-ce, build 60ccb22` and `docker-compose version 1.11.2, build dfed245`.
 
 If you are on Windows, go to docker settings and ensure the drive you are using is shared.
 
-This has been tested to work with at least `docker 17.03.0-ce, build 60ccb22` and `docker-compose version 1.11.2, build dfed245`.
+**Installation Instructions**
 
-Instructions are as follows. Clone this repository:
+- Clone this repository:
+`git clone https://github.com/hackersatcambridge/hac-website.git`
+- Navigate to the hac-website directory that you've now cloned
+- Create an empty file named `.env` and save it*
+- Now run `docker-compose up`
 
-```bash
-git clone https://github.com/hackersatcambridge/hac-website.git
-```
-Navigate to the hac-website directory that you've now cloned. Create an empty file named `.env`.
-Save the file. Why did we have to do that? This file will later be used to store sensitive information like API keys and environment-specific information in here for our server to use.
+This will build the project run the web server at [`localhost:3000`](http://localhost:3000). It will also watch for changes to the files and rebuild as necessarily. You can stop this process at any time with `^C` (`ctrl` + `C`).
 
-Now run:
+**Subsequent runs**
 
-```bash
-docker-compose up
-```
+When you want to run the project again (and you will, it's great):  
+
+- If the Dockerfile has been changed, run `docker-compose build`
+- run `docker-compose up`
+
+\**Why did we have to do that?* The `.env` file will later be used to store sensitive information like API keys and environment-specific information in here for our server to use.
+
+## Development
+- ["Um. It looks like your HTML is written in Swift."](#writinghtml)
+- Writing stylesheets (with BEM)
+- Working with the container
+	- What `docker-compose` is doing
+	- Running unit tests
+	- Running other commands
+
+
+### *"Um. It looks like your HTML is written in Swift."*
+
+You may notice that we aren't using a templating library for rendering HTML. Inspired by the likes of [Elm](http://elm-lang.org/) and [React](https://facebook.github.io/react/), we've written a module _HaCTML_ for making HTML type-safe and Swifty! For an example, have a look at our [home page](Sources/HaCWebsiteLib/views/home.swift).
+
+We're currently experimenting with the API for HaCTML, and once it's a little more stable we hope to release it as a standalone module.
 
-This will build all the files and run the web server with appropriate output. It will also watch for changes to the files and rebuild as necessarily. You can stop this process at any time with `^C` (`ctrl` + `C`).
+### Our Stylesheets and Their BEM
```

Comment:
I feel like this paragraph is duplicating information that is already available elsewhere. Keeping canonical resources for information are useful as you only have one source of truth. While the BEM website isn't great as an introduction, the [naming section on the AirBnB style guide](https://github.com/airbnb/css#oocss-and-bem) pretty much says the same thing. 

We could also say straight up that we're following this style guide (because at this stage, our codebase is)